enterprise/javaee.wildfly/nbproject/project.properties

```diff
@@ -21,7 +21,8 @@ auxiliary.de-markiewb-netbeans-plugins-eclipse-formatter.eclipseFormatterLocatio
 auxiliary.de-markiewb-netbeans-plugins-eclipse-formatter.enableFormatAsSaveAction=false
 auxiliary.de-markiewb-netbeans-plugins-eclipse-formatter.showNotifications=true
 auxiliary.de-markiewb-netbeans-plugins-eclipse-formatter.useProjectSettings=false
-javac.source=1.8
+javac.source=11
+javac.target=11
```

Comment:
Needed? From my POV we should keep changes to `javac.*` to a minimum and only bump when needed.

Reply:
It is not needed. Reverting and then merging.